docker/postgres-setup/init.sql

```diff
@@ -0,0 +1,32 @@
+-- create datahub database
+CREATE DATABASE IF NOT EXISTS DATAHUB_DB_NAME;
+USE DATAHUB_DB_NAME;
+
+-- create metadata aspect table
+CREATE TABLE metadata_aspect_v2 (
```

Comment:
Can we make this CREATE TABLE IF NOT EXISTS ?

Reply:
eecaf71